op-e2e/faultproof_test.go

```diff
@@ -231,6 +231,10 @@ func TestCannonDisputeGame(t *testing.T) {
 			sys.TimeTravelClock.AdvanceTime(game.GameDuration(ctx))
 			require.NoError(t, wait.ForNextBlock(ctx, l1Client))
 
+			// Manually resolve the game since the "honest actor" is the defender
+			// and won't call resolve.
+			game.Resolve(ctx)
```

Comment:
Nice that we have to manually resolve here. One callout is that I don't think we are testing doing the resolve now (though I could miss where we are doing it).

Reply:
This seems wrong to me - I'd have expected the challenger to resolve this game as it should be winning.  The defender is the actor who starts the game and provides the root claim (they are defending the root claim).